### PR TITLE
Check minimum required php version

### LINKED ISSUE #3501: Rector should display a clear error message when running with PHP < 7.2
# Feature Request

## Suggestion:
Rector should display a clear error message when the minimum required PHP version is not met.

## Problem to be solved:
Through the use of [Drupal Rector ](https://github.com/palantirnet/drupal-rector) we've seen many instances where people using Docker that has PHP 7.1

They install Rector **outside** of Docker (on their machine they have PHP>=7.2 and Composer wouldn't have any problem), but then they run Rector **inside** Docker, where they have PHP 7.1, and getting this error:
```
PHP Fatal error:  Declaration of _HumbugBox3630ef99eac4\Symfony\Component\HttpKernel\Config\FileLocator::locate($file, $currentPath = NULL, $first = true) must be compatible with _HumbugBox3630ef99eac4\Symfony\Component\Config\FileLocatorInterface::locate(string $name, ?string $currentPath = NULL, bool $first = true) in phar:///var/www/html/vendor/rector/rector-prefixed/rector/vendor/symfony/http-kernel/Config/FileLocator.php on line 20
Fatal error: Declaration of _HumbugBox3630ef99eac4\Symfony\Component\HttpKernel\Config\FileLocator::locate($file, $currentPath = NULL, $first = true) must be compatible with _HumbugBox3630ef99eac4\Symfony\Component\Config\FileLocatorInterface::locate(string $name, ?string $currentPath = NULL, bool $first = true) in phar:///var/www/html/vendor/rector/rector-prefixed/rector/vendor/symfony/http-kernel/Config/FileLocator.php on line 20
```

### PR DESCRIPTION
**Pull request by shaal**
Fixes #3501

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Damn. I revieved it instantly, but Github didn't send my review on mobile :cry: 
Sorry for noticing just now. It's there

File: bin/rector [hunk 1]
```diff
@@ -30,6 +31,8 @@ $autoloadIncluder->includeDependencyOrRepositoryVendorAutoloadIfExists();
 $autoloadIncluder->autoloadFromCommandLine();
 
 try {
+    checkMinimumPHPVersion();
```
Comment (TomasVotruba) [inline bin/rector:34]: Better use class, functions are very hard to autoload and re-use. Hell to maintain and decouple

**Comment by TomasVotruba** [comment]
How can I hep you to push this to merge?

**Comment by shaal** [comment]
I think we should close this, now that https://github.com/rectorphp/rector/pull/3665 is merged?

**Pull request closed without merging**